logstash-core/lib/logstash/filter_delegator.rb

```diff
@@ -39,9 +39,11 @@ def config_name
     def multi_filter(events)
       @metric_events.increment(:in, events.size)
 
-      clock = @metric_events.time(:duration_in_millis)
+      start_time = java.lang.System.current_time_millis
       new_events = @filter.multi_filter(events)
-      clock.stop
+      @metric_events.report_time(
+        :duration_in_millis, java.lang.System.current_time_millis - start_time
```

Comment:
@jordansissel hmm on second thought, just as a public utility on `Metric` then? It seems to me we're just making this harder to read.
I can see the point of having a method for this when the code was:

```rb
MILLISECONDS = 1_000.0.freeze
execution_time = (MILLISECONDS * (Time.now - @start_time)).to_i
```

But now it's literally just the standard JDK call for the *nix time stamp and taking a difference. IMO this makes it even harder to read in Ruby code since Ruby APIs seem to use `s` as a unit a lot and I always have to double check if it's `s` or `ms` to be sure. Here there's no ambiguity?
Maybe just keep this?